README.md

```diff
@@ -51,6 +53,9 @@ Physical path formats supported by guard tool:-
 * /sys-0/node-0/proc-0/mc-0/mi-0/mcc-0
 * sys-0/node-0/proc-0/mc-0/mi-0/mcc-0
 
+**Note:** clearresolvedrecord option will delete the oldest/first resolved
```

Comment:
This option did not implement in this patch. - You forget remove?

Reply:
Yes forgot to remove this from README.